osgi.enroute.web.simple.provider/src/osgi/enroute/web/server/provider/EnrouteApplicationIndexServer.java

```diff
@@ -0,0 +1,165 @@
+package osgi.enroute.web.server.provider;
+
+import java.net.*;
+import java.util.*;
+
+import javax.servlet.http.*;
+
+import org.osgi.dto.*;
+import org.osgi.framework.*;
+import org.osgi.service.component.annotations.*;
+import org.osgi.service.log.*;
+import org.osgi.util.tracker.*;
+
+import aQute.lib.io.*;
+import aQute.lib.json.*;
+import aQute.libg.sed.*;
+import osgi.enroute.servlet.api.*;
+import osgi.enroute.web.server.cache.*;
+import osgi.enroute.web.server.config.*;
+import osgi.enroute.web.server.exceptions.*;
+import osgi.enroute.web.server.provider.EnrouteApplicationIndexServer.IndexDTO.*;
+
+@Component(
+		service = { ConditionalServlet.class }, 
+		immediate = true, 
+		property = {
+				"service.ranking:Integer=1001", 
+				"name=" + EnrouteApplicationIndexServer.NAME, 
+		}, 
+		name = EnrouteApplicationIndexServer.NAME, 
+		configurationPid = BundleMixinServer.NAME,
+		configurationPolicy = ConfigurationPolicy.OPTIONAL)
+public class EnrouteApplicationIndexServer implements ConditionalServlet {
+
+	static final String NAME = "osgi.enroute.simple.web.application";
+
+	static class IndexDTO extends DTO {
+
+		List<ApplicationDTO>	applications	= new ArrayList<>();
+		Map<String,Object> configuration;
+		
+		static class ApplicationDTO extends DTO {
```

Comment:
must be public and probably better to move it one level up so it is a peer of the IndexDTO